projects/ngneat/cashew/src/lib/httpCacheInterceptor.module.ts

```diff
@@ -8,15 +8,20 @@ import { HttpCacheManager } from './httpCacheManager.service';
 import { DefaultTTLManager, TTLManager } from './ttlManager';
 import { DefaultHttpCacheGuard, HttpCacheGuard } from './httpCacheGuard';
 import { RequestsQueue } from './requestsQueue';
+import { HttpCacheLocalStorage } from './httpCacheLocalStorage';
 
 @NgModule({})
 export class HttpCacheInterceptorModule {
   static forRoot(config: Partial<HttpCacheConfig> = {}): ModuleWithProviders {
+    const conf = mergeConfig(config);
     return {
       providers: [
-        { provide: HTTP_CACHE_CONFIG, useValue: mergeConfig(config) },
+        { provide: HTTP_CACHE_CONFIG, useValue: conf },
         { provide: KeySerializer, useClass: DefaultKeySerializer },
-        { provide: HttpCacheStorage, useClass: DefaultHttpCacheStorage },
+        {
```

Comment:
This approach isn't **treeshakeable**. We need to expose the provider and the users can use it if they want:

```ts
{
  provide: HttpCacheStorage,
  useClass: HttpCacheLocalStorage
}
```